beacon-chain/blockchain/service.go

```diff
@@ -334,7 +346,10 @@ func (c *ChainService) blockProcessing(done <-chan struct{}) {
 			activeState := c.chain.ActiveState()
 			crystallizedState := c.chain.CrystallizedState()
 			if isTransition {
-				crystallizedState, activeState = c.chain.initCycle(crystallizedState, activeState)
+				crystallizedState, activeState, err = c.chain.initCycle(crystallizedState, activeState)
```

Comment:
This block is untested. Can you test this isTransition = true block?

Reply:
Done! thanks for catch